resources/api/resource.py

```diff
@@ -336,7 +336,9 @@ def __init__(self, *args, **kwargs):
             ('unit__name_fi', 'unit_name_fi'),
             ('unit__name_en', 'unit_name_en'),
             ('unit__name_sv', 'unit_name_sv'),
-            ('type', 'type'),
```

Comment:
You should not modify or remove any already existing fields to maintain backwards compatibility.

Reply:
@tuomas777 no implementations on the frontend have been made yet, it's a brand new feature.

Reply:
Ok, so that has not been even deployed yet? In that case it is probably ok :)

Reply:
Changes like these should be discussed before they are made. In this case the change can be made since it is only deployed on the test server. If this feature had made it into production, we could not change it anymore. Also, Respa exists independent of Varaamo and has other clients, so whether or not it is implemented in Varaamo does not matter.